src/freenet/clients/fcp/RequestStatus.java

```diff
@@ -63,10 +69,15 @@ synchronized void restart(boolean started) {
 		this.totalBlocks = total;
 		this.minBlocks = min;
 		this.fetchedBlocks = fetched;
+        // clone() because Date is mutable
```

Comment:
What's your reasoning for including this comment? It's certainly the right behavior, but I don't think it's remarkable enough to warrant a comment.